src/components/ciPipeline/ciPipeline.css

```diff
@@ -193,14 +193,335 @@
+}
+
+.configuration-container .right-radius input + .radio__item-label {
+    border-radius: 0 4px 4px 0 !important;
+    border-left: 0px;
+}
+
+.radio-group input[type='checkbox']:checked + .radio__item-label {
+    background: var(--N100);
+    color: var(--N900);
+}
+
+.plugin-card:hover {
+    box-shadow: 0px 0px 4px 1px var(--N200);
+    border-radius: 4px;
+}
+
+.tag-container {
+    padding: 0px 4px;
+    text-transform: capitalize;
+}
+
+.lh-16 {
+    line-height: 16px;
+}
+
+.pc-icon-container {
+    width: 40px;
+    height: 40px;
+    justify-content: center;
+    align-items: center;
+}
+
+.row-container {
+    display: grid;
+    grid-template-columns: 220px auto;
+}
+
+.mount-row {
+    display: grid;
+    grid-template-columns: 200px auto 28px auto 30px;
+}
+
+.label-width {
+    width: 250px;
+}
+
+.justify-start {
+    justify-content: flex-start;
+    padding-left: 0;
+}
+
+.variable-container {
+    display: grid;
+    grid-row-gap: 8px;
+    grid-template-columns: 200px 100px auto;
+}
+
+.custom-variable-container {
+    display: grid;
+    position: relative;
+}
+
+.variable-equal-icon {
+    position: absolute;
+    left: -37px;
+    top: 16px;
+}
+
+.condition-container {
+    display: grid;
+    grid-row-gap: 8px;
+    grid-template-columns: 80px auto 70px auto 30px;
+}
+
+.custom-input__port-map {
+    display: grid;
+    grid-template-columns: 80px 28px 80px 30px;
+}
+
+.custom-script__output-directory {
+    display: grid;
+    grid-template-columns: auto 30px;
+}
+
+.content-fit {
+    width: fit-content;
+}
+
+.justify-space {
+    justify-content: space-between;
+}
+
+.border-bottom-1-n2 {
+    border-bottom: 1px solid var(--N200);
+}
+
+.active-tab {
+    border-bottom: 2px solid var(--N900);
+}
+
+.row-container .radio-group {
+    justify-content: flex-start;
+}
+
+.row-container .form__radio-item {
+    min-width: 0px !important;
+}
+
+.pl-200 {
+    padding-left: 200px;
+}
+
+.bottom-radius-4 {
+    border-bottom-left-radius: 4px;
+    border-bottom-right-radius: 4px;
+}
+
+.top-radius-4 {
+    border-top-left-radius: 4px;
+    border-top-right-radius: 4px;
+}
+
+.left-radius-4 {
+    border-top-left-radius: 4px;
+    border-bottom-left-radius: 4px;
+}
+
+.right-radius-4 {
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
+}
+
+.no-top-border {
+    border-top: none;
+}
+
+.vertical-line-connector {
+    border-left: 1px solid var(--N200);
+    height: 12px;
+    position: relative;
+    left: 50%;
+    width: 5px;
+}
+
+.variable-description {
+    padding: 6px 10px;
+    width: 180px;
+    background: var(--N900);
+    border: 1px solid var(--N200);
+    border-radius: 4px;
+}
+
+.no-top-border {
+    border-top: 0px;
+}
+
+.no-bottom-border {
+    border-bottom: 0px;
+}
+
+.no-left-border {
+    border-left: 0px;
+}
+
+.top-left-radius {
+    border-top-left-radius: 4px;
+}
+
+.pl-220 {
+    padding-left: 220px;
+}
+
+.h-32 {
+    height: 32px;
+}
```

Comment:
The same goes for heights. We can have some standard heights like 32px, 36px, 48px that are being used defined in base.scss.